src/shorthands/size.js

```diff
@@ -21,7 +21,7 @@
  * }
  */
 
-function size(height: string, width: string = height) {
+function size(height: string | number, width: string | number = height) {
```

Comment:
Unitless height/width is invalid unless 0, so we should put the same `=== 0` check here as in the other modules.